modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/system/test/ObjectDefinitionLabelKeyProviderTest.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.system.test;
```

Comment:
@robertoDiaz you only need to update the test package to `com.liferay.object.label.key.provider.test`. 

But the code looks good!